legislative/Q28223935/2018-01-01-to-2018-12-31/popolo-m17n.json

```diff
@@ -55,7 +55,7 @@
         }
       ],
       "associated_wikidata_positions": [
-        "Q132050"
+        "Q1571023"
       ],
```

Comment:
Similar question as above: are we expecting state level data (parent) to be stored as part of each LGA (child)?

Reply:
Yep, as above :-)